docs/developer-guide/lighting.md

```diff
@@ -0,0 +1,42 @@
+# Lighting
+
+Luma.gl supports different shading modules.
+To make 
+
+
+## Light Sources
+
+## Example
+You can use the `lighting` example to play with lights.
+
+## Currently supported lighting models
+- Phong (Available in the `phong-lighting` module)
+- PBR (Available in the `pbr` module)
```

Comment:
better to link the Phong and PBR module folder or code file, and remove the comment in ()